LarryKarani/iReporterApi
# #162424061 user signup

__What does this PR do?__
  allow users to signup

__Description of Task to be completed?__
- Create a  signup view that allows users to create an account
- hash users password before storing in the database


__How should this be manually tested?__
- clone the repo
- create a virtualenv and install the project's dependencies with pip install -r requirements.txt
run the server
- open postman and navigate to api/v2/auth/signup
- enter the signup details

__Any background context you want to provide?__
- the phone number should not be less than 10 characters
What are the relevant pivotal tracker stories?
##162424061
